build.gradle

```diff
@@ -75,7 +82,7 @@ dependencies {
     implementation 'javax.servlet:javax.servlet-api:3.1.0'
     implementation 'commons-codec:commons-codec:1.15'
     implementation 'commons-io:commons-io:2.11.0'
-    implementation ("org.apache.camel:camel-spring-redis:3.12.0")
+    implementation("org.apache.camel:camel-spring-redis:3.12.0")
```

Comment:
Why are we adding this Camel dependency? 
The goal is to make the existing service Camel-free.